lib/kvdb/ikvdb.c

```diff
@@ -383,20 +380,17 @@ ikvdb_create(const char *kvdb_home, struct kvdb_cparams *params)
     wal_destroy(mp, meta.km_wal.oid1, meta.km_wal.oid2);
 cndb_cleanup:
     cndb_destroy(mp, meta.km_cndb.oid1, meta.km_cndb.oid2);
-mpool_cleanup:
-    {
-        struct mpool_dparams mp_dparams;
-
-        for (int i = 0; i < HSE_MCLASS_COUNT; i++) {
-            if (params->storage.mclass[i].path[0] != '\0') {
-                strlcpy(
-                    mp_dparams.mclass[i].path,
-                    params->storage.mclass[i].path,
+mpool_cleanup : {
```

Comment:
This is really strange. I wonder if this would happen on clang-format v15

Reply:
It is odd.  v15 does the same.

Reply:
will not be addressed.